### PR TITLE
Allow evaluation of lists/tuples/sets/dicts

### PR DESCRIPTION
**Pull request by pwwang**
#5

### CODE REVIEW

File: tests/test_core.py [hunk 1]
```diff
@@ -31,8 +31,8 @@ def test_eval_names():
     x = 3
     check_eval(
         "(x, check_eval, len), nonexistent",
-        x, check_eval, len,
-        # TODO tuple (x, check_eval, len),
+        x, check_eval, len, (x, check_eval, len)
+        # Now tuple is recognized
```
Comment (alexmojaki) [inline tests/test_core.py:35]:
```suggestion
```

**Comment by alexmojaki** [comment]
Thanks for the PR! The tests are good, the implementation is a good start, but it has a decent amount of duplicated logic and duplicated computation. Can you refactor it a bit?

**Comment by alexmojaki** [comment]
The code you added to `_handle`, considered in isolation

**Comment by pwwang** [comment]
I am actually not quite sure if this is what you wanted it to be...

**Comment by pwwang** [comment]
This isn't the first time that travis gets stuck. You probably want to switch to GH actions ...

**Comment by alexmojaki** [comment]
Indeed, I recently did that for the first time with `executing` because Travis is going through some changes. But it's still enough of a pain that I'm going to procrastinate it. Looks like travis woke up in the end. Thanks for the PR!

**Pull request merged**